app/src/main/res/values/strings.xml

```diff
@@ -111,4 +111,10 @@
     <string name="symbol_percent"> %</string>
     <string name="title_post_detail">%1$s Post</string>
     <string name="title_selected_image_deital">Image detail</string>
+    <string name="title_no_media">00:00</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_no_media` appears to be unused
 Unused resources make applications larger and slow down builds.